src/components/SearchBlock/style.css

```diff
@@ -29,7 +29,7 @@
 
 .ant-select-selection, .ant-select-selection--single {
   width: 130px;
-  height: 36px;
+  height: 36px !important;
```

Comment:
Add a comment with the reason of using !important. For example, 'library styles override'